vendor/golang.org/x/sys/unix/syscall_darwin.1_12.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 The Go Authors. All rights reserved.
```

Comment:
Are all these "vendor" files intentionally uploaded?

Reply:
Yes, this way we don't break the code if certain versions are not available anymore. We had that problem on Eunomia and @seanmalloy moved all other project over to this way of handling it. I want to stay with the theme.